scripts/quests/sandoria/PLD_AF1_Sharpening_The_Sword.lua

```diff
@@ -0,0 +1,160 @@
+-----------------------------------
+-- Sharpening The Sword PLD AF 1
+-- !addquest 0 90
+-----------------------------------
+-- Ailbeche: !gotoid 17723401
+-- Sobane: !gotoid 17719322
+-- Stalagmite: !gotoid 17568175
+-----------------------------------
+require('scripts/globals/items')
+require("scripts/globals/keyitems")
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require("scripts/globals/status")
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local ordellesID = require("scripts/zones/Ordelles_Caves/IDs")
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.SHARPENING_THE_SWORD)
+
+quest.reward =
+{
+    item     = xi.items.HONOR_SWORD,
+    fame     = 30,
+    fameArea = xi.quest.fame_area.SANDORIA,
+}
+
+quest.sections =
+{
+    -- Section: Quest available.
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getMainJob() == xi.job.PLD and
+                player:getMainLvl() >= xi.settings.main.AF1_QUEST_LEVEL and
+                player:getQuestStatus(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.FATHER_AND_SON) == QUEST_COMPLETED and
+                player:getCharVar("Quest[0][4]Prog") == 0 -- must have turned the willow fishing rod back
+        end,
+
+        [xi.zone.NORTHERN_SAN_DORIA] =
+        {
+            ['Ailbeche'] =
+            {
+                onTrigger = function(player, npc)
+                    if player:getCharVar("Quest[0][90]Prog") == 0 then
+                        return quest:progressEvent(45) -- Starts Quest.
+                    elseif player:getCharVar("Quest[0][90]Prog") == 1 then --declined first
```

Comment:
Strange that you're getting prog for this quest like this! lol you can just use the following, no?
```suggestion
                    elseif quest:getVar(player, 'Prog') == 1 then --declined first
```